content/en/real_user_monitoring/installation/_index.md

```diff
@@ -64,10 +102,22 @@ Paste the generated code snippet into the head tag (in front of any other script
 
 **Note**: The `window.DD_RUM` check is used to prevent issues if a loading failure occurs with the library.
 
+## Initialization parameters
+
+| Parameter            | Type   | Required | Default | Description                                                                                                  |
+|----------------------|--------|----------|---------|--------------------------------------------------------------------------------------------------------------|
+| `applicationId`      | String | Yes      | ``      | The RUM application ID.                                                                                      |
+| `clientToken`        | String | Yes      | ``      | A [Datadog Client Token][5].                                                                                 |
+| `datacenter`         | String | Yes      | `us`    | The Datadog Site of your organization. `us` for Datadog US site, `eu` for Datadog EU site.                   |
+| `resourceSampleRate` | Number | No       | `100`   | Percentage of tracked sessions with resources collection. `100` for all, `0` for none of them.               |
```

Comment:
Do we still want to document `resourceSampleRate` then?

Reply:
let's just mention it as it's available but not expose it too much.